gensim/test/test_summarization.py

```diff
@@ -128,3 +128,14 @@ def test_corpus_summarization_ratio(self):
             expected_summary_length = int(len(corpus) * ratio)
 
             self.assertEqual(len(selected_docs), expected_summary_length)
+
+    def test_keywords_runs(self):
+        pre_path = os.path.join(os.path.dirname(__file__), 'test_data')
+
+        with utils.smart_open(os.path.join(pre_path, "mihalcea_tarau.txt"), mode="r") as f:
+            text = f.read()
+
+        kwds = keywords(text)
```

Comment:
Sure, I also put some contrived `assertTrue` statements in there since it doesn't feel like a *real* test without asserts somewhere. If you think it's fine to simply call the `keywords` function though I can take them out.